### PR TITLE
Feature/book pages

### LINKED ISSUE #14: Play screen resizing proportionally within frame fix

### LINKED ISSUE #19: "Close Book" button needs to match the previous URL path to return the user to the correct portion of the story

### PR DESCRIPTION
**Pull request by qrispi**
Change Summary:
- This merge completes functionality for the book and also positions all content within the frame better
- A user can now view random cat facts in the book, bookmark them, unbookmark them and view their bookmarks
- Bookmarks persist across the duration of the story
- All buttons conditionally render to only be displayed if functional

Functions/Methods:
- turnPage: increments the page number for the Page component to facilitate in displaying the correct fact, rendering the buttons and rendering the header
- removeFact: reducer action that removes a saved fact from the store (removes the bookmark)

Closes #19, Closes #14

### CODE REVIEW

File: src/components/Page/Page.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    const turnPage = (direction) => {
+      setPageNum(pageNum + direction)
+    }
+    
+    return (
+    <>
+      <h2>Page {pageNum + 1}</h2>
+      <p>{savedFacts[pageNum]}</p>
+      {!savedFacts[pageNum] && <p>Oh no! You ripped out this page! Use the buttons to go to a different page.</p>}
```
Comment (BriBourassa) [inline src/components/Page/Page.js:20]: Love that deleting equates to ripping out a page - good visual!

**Review by BriBourassa — APPROVED** [review]
Absolutely amazing, nice work!

File: src/components/Page/Page.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      <NavLink to="/book">
+        <button>BACK</button>
+      </NavLink>
+      {pageNum > 0 && <button onClick={() => turnPage(-1)}>Prev Page</button>}
+      {savedFacts.includes(savedFacts[pageNum]) && <button onClick={() => dispatch(removeFact(savedFacts[pageNum]))}>Remove Bookmark</button>}
+      {savedFacts.length >= (pageNum + 2) && <button onClick={() => turnPage(1)}>Next Page</button>}
```
Comment (jsahim) [inline src/components/Page/Page.js:26]: amazing logic!

**Review by jsahim — APPROVED** [review]
look good!

**Pull request merged**